crates/cargo-builder/src/package.rs

```diff
@@ -33,6 +33,7 @@ impl PackageInfo {
         let metadata = MetadataCommand::new()
             .manifest_path(&current_project)
             .features(CargoOpt::AllFeatures)
+            .verbose(true)
```

Comment:
Is it enough? It affects only `cargo metadata` command, not `cargo build`

Reply:
`cargo build` was already giving output, it was just `metadata` command that was not.